doc/schemas/node_openapi.json

```diff
@@ -277,6 +277,7 @@
       "NodeStatus": {
         "enum": [
           "Pending",
+          "CatchingUp",
```

Comment:
I'm a bit worried about this naming, Trusted formerly implied two things:
  1. that the node was participating in replication and election quorums
  2. that it had received a copy of the ledger secrets (ie. trusted from a confidentiality perspective).

`CatchingUp` effectively corresponds to the latter but not the former, but the name doesn't convey it strongly, and the existence of a separate `Trusted` stage even suggests otherwise.

I propose:

`CatchingUp` -> `Replicating`
`Trusted` -> `Participating`

or:

`CatchingUp` -> `Learner`
`Trusted` -> `Replica`

Reply:
I agree, we need to change that. To me, "Participating" sounds a bit too weak, but maybe it would be a good thing to remove the connotation with confidentiality. Maybe Replicating/Replica? Or Cloning/Replica?